ui/packages/consul-ui/app/components/topology-metrics/stats/index.scss

```diff
@@ -1,10 +1,11 @@
-.stats {
+.topology-metrics-stats {
   padding: 12px 12px 0 12px;
   display: flex;
   flex-flow: row wrap;
   justify-content: space-between;
   align-items: stretch;
   width: 100%;
+  border-top: 1px solid $gray-200;
```

Comment:
I'll mention this bit ^ in the approval, just dropped this here so you can see easily

Reply:
👍 updated